package.json

```
@@ -19,6 +19,7 @@
     "prod": "NODE_ENV=production webpack serve --config config/dev.webpack.config.js",
     "server:ctr": "node src/server/generateServerKey.js",
     "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
+    "start:proxy": "NODE_ENV=development USEPROXY=true webpack serve --config config/dev.webpack.config.js",
```

It would probably be better to use the prod proxy here
```suggestion
    "start:proxy": "NODE_ENV=development API_ENDOINT=https://cloud.redhat.com webpack serve --config config/dev.webpack.config.js",
```

---

okie dokie, relied on api_endpoint as  useproxy flag, start:proxy now by default runs prod, `API_ENDOINT=https://ci.cloud.redhat.com npm run start` would be a custom proxy